packages/sui-mono/src/config.js

```diff
@@ -35,6 +36,15 @@ module.exports = {
   },
   getPublishAccess: function () {
     return publishAccess
+  },
+  getProjectName: function () {
+    return projectPackage.name
+  },
+  isMonoPackage: function () {
+    const folders = cwds(path.join(basePath, packagesFolder), deepLevel)
+    const scopePath = folders.pop()
+
```

Comment:
I don't have a better idea, but if it only uses the first folder to define when all the project is mono-package, it may be problematic.

But I don't have a better idea either. What do you think ?!